crawler/attribution/entrypoints/blob-url.js

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
why do you have brackets on all those files? To prevent const/let leaking to the global context?

Reply:
yep, exactly.